driver/Cargo.toml

```diff
@@ -21,7 +21,7 @@ rustc-hex = "*"
 sha2 = "*"
 byteorder = "*"
 hex = "0.3.2"
-web3 = { git = "https://github.com/tomusdrw/rust-web3", rev = "56ad08e" }
+web3 = "*"
```

Comment:
Let's 📌 to a specific version here. Otherwise we might get random failures when the library pushes a new version that is breaking in some way. I think we should control the upgrades ourselves.

Reply:
Doesn't pinning ourselves to a specific version this early in development mean that we might get stuck behind and miss updates?

Reply:
I'm not saying we should never update, but we should be in control of our update process, specifically if the dependent library seems to be unstable in that it introduces breaking changes on a regular basis.